build/Build.md

```diff
@@ -0,0 +1,190 @@
+# 1. Building a container image
+
+This chapter depicts important information about building your first WSO2 container image.
+
+### Contents
+
+The following will be discussed in detail in the document.
+
+* [1.1. Why should you choose Docker as your container runtime ?](#Why-docker)
+* [1.2. Using WSO2 images](#using-wso2-images)
+* [1.3. Building from source](#building-from-source)
+* [1.4. Tips to harden your image](#tips-to-harden-your-image)
+* [1.5. Tips on Tagging](#tips-on-tagging)
+* [1.6. Vulnerability scanning and container security check](#scanning)
+* [1.7. Image signing and Verification](#signing)
+
+<br/>
+
+## 1.1. Why should you choose Docker as your container runtime ?
+
+Started just 7 years ago in 2013, Docker introduced what would become the industry standard for containers and is undeniably yet the defacto standard to build, share and run containerized apps. 
+
+<img src="../imgs/build/docker-usage.jpg" width="550">
+
+Among many factors which led WSO2 to choose and recommend Docker as its first choice for a container runtime, following is key.
+
+* Docker’s ability to manage a complete container lifecycle.
+* Less learning curve and advanced feature set.
+* Wide community support towards its development and maintenance.
+* High popularity and customer demand.
+
+<br/>
+<img src="../imgs/build/docker-adoption.png" width="900">
+
+
+## 1.2. Using WSO2 images
+
+WSO2-curated product Docker images are available for usage from both :
+
+[1] [Docker Hub](https://hub.docker.com/u/wso2/) : <br/>
+* Contains images built with the inclusion of GA released product packs and is publicly available for use free of charge.
+
+[2] [WSO2 Private Docker Registry, docker.wso2.com](https://docker.wso2.com/) : <br/>
+* Contains docker images built with the inclusion of regularly updated product packs and is accessible only for its subscribed customers. 
```

Comment:
@DilanUA IMO it would be ideal if we could find a way to point to details regarding the [WSO2 Subscription](https://wso2.com/subscription/)?